CONTRIBUTING.rst

```diff
@@ -0,0 +1,7 @@
+============
+Contributing
+============
+
+Contributions are welcome, and they are greatly appreciated! Every
+little bit helps, and credit will always be given.
+
```

Comment:
Could link to some guides about setting up a python development environment?

Reply:
added link to sunpy newcomer guide.